src/pages/components/tabs/usage.mdx

```diff
@@ -410,6 +415,12 @@ label.
 </Column>
 </Row>
 
+<InlineNotification>
+  Note: Tabs with an icon is an example of a pattern. It is not currently
+  implemented by default in the component, it is instead enabled through the
+  composable component API. This requires additional styling and configuration.
+</InlineNotification>
```

Comment:
Oops I suggested the wrong note to be an inline notifications. This one is just a caption

```suggestion
<Caption>
  Note: Tabs with an icon is an example of a pattern. It is not currently
  implemented by default in the component, it is instead enabled through the
  composable component API. This requires additional styling and configuration.
</Caption>
```